tensorflow/g3doc/api_docs/python/array_ops.md

```diff
@@ -3135,8 +3135,6 @@ Compute gradients for a FakeQuantWithMinMaxVarsPerChannel operation.
 ## Other Functions and Classes
 - - -
 
-### `tf.concat_v2(values, axis, name='concat_v2')` {#concat_v2}
```

Comment:
This is an autogenerated file.
Please revert, I will rerun doc generation after cherrypicks.